src/testdata/urlpatterntestdata.json

```diff
@@ -2787,7 +2782,7 @@
     }
   },
   {
-    "skip": "ingoreCase is not yet implemented",
+    "skip": "this does not error. same happens in Chrome as well.",
```

Comment:
This is a bit strange, but not sure what is up with it.

Reply:
The test passes on Chrome in WPT: 
![image](https://github.com/user-attachments/assets/8b82f900-073e-467e-b001-c0ef998a6d26)